bzl/repositories.bzl

```diff
@@ -66,6 +66,24 @@ def pcl_repositories():
         urls = ["https://github.com/qhull/qhull/archive/2019.1.tar.gz"],
     )
 
+    _maybe_repo(
+        http_archive,
+        name = "flann",
+        build_file = "@rules_pcl//third_party:flann.BUILD",
+        sha256 = "dfbb9321b0d687626a644c70872a2c540b16200e7f4c7bd72f91ae032f445c08",
+        strip_prefix = "flann-1.8.4-src",
+        urls = ["https://www.cs.ubc.ca/research/flann/uploads/FLANN/flann-1.8.4-src.zip"],
+    )
+
+    _maybe_repo(
+        http_archive,
+        name = "lz4",
+        build_file = "@rules_pcl//third_party:lz4.BUILD",
```

Comment:
Did you forget to add this file? I don't see lz4.BUILD in the third_party directory